freud/density.pyx

```diff
@@ -301,6 +300,115 @@ cdef class GaussianDensity(_Compute):
             return None
 
 
+cdef class SphereVoxelization(_Compute):
+    R"""Computes a grid of voxels occupied by spheres.
+
+    This class constructs a grid of voxels. From a given set of points and a
+    desired radius, a set of spheres are created. The voxels are assigned a
+    value of 1 if their center is contained in one or more spheres and 0
+    otherwise. The dimensions of the grid are set in the constructor, and can
+    either be set equally for all dimensions or for each dimension
+    independently.
+
+    Args:
+        width (int or Sequence[int]):
+            The number of bins to make the grid in each dimension (identical
+            in all dimensions if a single integer value is provided).
+        r_max (float):
+            Sphere radius.
+    """  # noqa: E501
```

Comment:
I don't think there's any reason to request longer lines here, usually we only do this in the docstrings for classes where the argument type involves a long `:class:numpy.ndarray...` type (or similar) that overflows on a single line.
```suggestion
    """
```